src/Microsoft.Tye.Hosting/TransformProjectsIntoContainers.cs

```diff
@@ -83,10 +83,9 @@ private async Task TransformProjectToContainer(Service service, ProjectRunInfo p
             serviceDescription.RunInfo = dockerRunInfo;
         }
 
-        private static string DetermineContainerImage(string targetFramework)
+        private static string DetermineContainerImage(ProjectRunInfo project)
         {
-            // TODO: Determine the base image from the tfm
-            return "mcr.microsoft.com/dotnet/core/sdk:3.1-buster";
+            return $"mcr.microsoft.com/dotnet/core/sdk:{project.TargetFrameworkVersion}";
```

Comment:
This pattern would assume that all development, if using Linux containers, would be done on Debian based containers given the TFM base image is always such. How would we specify an alpine or other flavor? Should we consider looking leveraging  [Container MS Build Properties](https://docs.microsoft.com/en-us/visualstudio/containers/container-msbuild-properties?view=vs-2019) like `DockerfileTag` here?

Reply:
Not possible to override it today. I’d imagine we’ll eventually let you control the base image or docker file completely

Reply:
Seems reasonable to support these in msbuild for alignment anyways.

Reply:
@spboyer can you file an issue for this?

Reply:
Done - https://github.com/dotnet/tye/issues/240